5_databases/week3/Reservation.sql

```diff
@@ -0,0 +1,29 @@
+use homwwork3;
+--1--
+SELECT*
+FROM `Reservation`;
+--2--
+INSERT into Reservation (id, number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email)
+VALUES(1, 10, 2, '2023-09-03', '+4528382639', 'nada_jrad', 'ada@gmail.com');
+INSERT into Reservation (id, number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email)
+VALUES(2, 12, 3, '2023-12-02', '+4528452639', 'jannah', 'jannah@gmail.com');
+INSERT into Reservation (id, number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email)
+VALUES(3, 12, 3, '2023-12-12', '+4528452639', 'jannah', 'jannah@gmail.com');
```

Comment:
You normally don't want to provide the `AUTO_INCREMENT` values like `id`. Instead omit that and let the database figure out the next free id and automatically assign it to your newly inserted row.

```sql
INSERT INTO Reservation (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email)
VALUES(10, 2, '2023-09-03', '+4528382639', 'nada_jrad', 'ada@gmail.com');
```